WordPress/UITestsFoundation/Screens/Login/Unified/PasswordScreen.swift

```diff
@@ -38,6 +38,13 @@ public class PasswordScreen: ScreenObject {
         let continueButton = app.buttons["Continue Button"]
         continueButton.tap()
 
+        // The Simulator might as to save the password which, of course, we don't want to do
+        if app.buttons["Save Password"].waitForExistence(timeout: 5) {
```

Comment:
If we find no reasons as to why this happens, and we go with this change, does it make sense to reduce the timeout to `1` or `2`? I haven't seen the prompt, but I assume that it normally appears is less time than 5 seconds. What worries me is that this block will be used in almost every test on CI, meaning almost guaranteed (17 tests * 5 sec) 1.5 minutes increase in execution time.